components/Button/ButtonPrimary.tsx

```diff
@@ -0,0 +1,20 @@
+import Button from "./buttonPrimary.style";
+import Link from "next/link";
+import React from 'react'
+
+interface Buttons {
```

Comment:
Let's decide about the name of the typescript's types so it will be consistent throughout the project.
I suggest naming the propTypes of the component using the name of it and the Types. For example: interface ButtonTypes.